server/setup.cfg

```
@@ -1,12 +1,12 @@
 [metadata]
 name = weaverbird
-version = 0.12.6
+version = 0.13.0
 
 [options]
 package_dir =
   = src
 packages = find:
-python_requires = >=3.8
+python_requires = >=3.10, <3.11
```

```suggestion
python_requires = >=3.10
```
devrait être good non?

---

mouais... allez